README.md

```diff
@@ -1,20 +1,96 @@
 # ml-spectra-fitting
 
-[![NPM version][npm-image]][npm-url]
-[![build status][travis-image]][travis-url]
-[![npm download][download-image]][download-url]
+[![NPM version][npm-image]][npm-url] [![build status][travis-image]][travis-url] [![npm download][download-image]][download-url]
 
-Bell optmization library. It contains:
-
-- `optimizeSingleLorentzian`: Function to fit a curve to a single 3 parameter Lorentzian function
-- `optimizeLorentzianSum`: Function to fit a curve to a set of 3 parameters Lorentzian functions
-- `optimizeSingleGaussian`: Function to fit a curve to a single 3 parameter Gaussian function
-- `optimizeGaussianSum`: Function to fit a curve to a set of 3 parameters Gaussian functions
+Curve fitting method in javascript.
 
 ## Installation
 
 `$ npm install ml-spectra-fitting`
 
+## [API Documentation](https://mljs.github.io/spectra-fitting/)
+
+This is spectra fitting package that support gaussian, lorentzian and pseudoVoigt kind of shapes. It is a wrapper of [ml-levenberg-marquardt](https://github.com/mljs/levenberg-marquardt)
+
+## Example
+
+```js
+// import library
+import { optimizeSum, optimize } from "ml-spectra-fitting";
+// const { optimizeSum, optimize } = require('ml-levenberg-marquardt');
+
+let nbPoints = 31;
+let xFactor = 0.1;
+let x = new Float64Array(nbPoints);
+for (let i = 0; i < nbPoints; i++) {
+  x[i] = (i - nbPoints / 2) * xFactor;
+}
+// function that receives the parameters and returns
+// a function with the independent variable as a parameter
+function sumOfLorentzians(p) {
```

Comment:
Why is this function here ? If it is simply to create a function please use the 'spectrum-generator' in the example